app/src/main/java/hangbt/hust/androidbasic/MainActivity.java

```diff
@@ -3,12 +3,229 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
+    TextView tv0, tv1, tv2, tv3, tv4, tv5, tv6, tv7, tv8, tv9, tv00, tv_cong, tv_tru, tv_nhan, tv_chia, tv_bang;
```

Comment:
1. Để private khi không có nhu cầu truy cập từ bên ngoài 
2. Đặt tên biến cụ thể, có thể đọc được
3. Trong trường hợp có nhiều biến có cùng xử lý, mình có thể khai báo dưới dạng mảng ]
4. Đặt tên biến theo tiếng anh, không sử dụng tiếng việt

```suggestion
private TextView[] textNumber= new TextView[10];
private TextView textResult, textAdd, text   .... 
```
Đặt tên biến, code theo style của google : https://source.android.com/setup/contribute/code-style